core/src/main/java/com/yahoo/oak/OakUnsafeDirectBuffer.java

```diff
@@ -47,8 +47,7 @@
      *
      * @return the underlying ByteBuffer.
      */
-    ByteBuffer getByteBuffer();
-
+    ByteBuffer getByteBuffer(); //description change?
```

Comment:
Using reflection, can we create a smaller ByteBuffer? Not the entire ByteBuffer of the Block? If yes, this should be much preferable...

Reply:
done!
in this case we dont need to save the capacity of the allocated memory

Reply:
👍